docs/examples/README.md

```diff
@@ -1,52 +1,60 @@
-## BESKRIVELSE AV EKSEMPELDATA
+## DATASET EXAMPLES
 ______
 ### SYNT_PERSON_INNTEKT
-Dette datasettet inneholder syntetisk data om personer sin årlige inntekt.
-Strukturen av en rad er nærmere beskrevet i metadata, men kan summeres opp slik:
+This dataset contains synthetic data describing the yearly income of a group of persons.
+The structure of a row in the data is described well by the metadata, but simply put:
+        
+        PERSON_ID;INCOME;START;STOP
 
-        PERSON_ID;INNTEKT;START;STOP
-
-Temporalitetstypen på dette datasettet er "ACCUMULATED". Det betyr at når vi har en rad som,
+The temporality type of this dataset is "ACCUMULATED". Meaning that when we have a row:
 
         00000000000001;1000;2020-01-01;2020-12-31;
 
-betyr det at inntekten til en person(00000000000001) akkumulert fra starten til slutten av 2020 var 1000 kroner.
-Vi kan ha flere rader med samme person, men disse radene må beskrive forskjellige, ikke- overlappende tidsspenn for den personen.
+It means that the income of a person(00000000000001) accumulated through 2020 is 1000 NOK.
+We can have several rows for the same person, but these rows must describe different, non-intersecting timespans.
 &nbsp;
 
 &nbsp; 
 
-
-
 ### SYNT_PERSON_KJOENN
-Dette datasettet innholder info om kjønn til en befolkning.
-Strukturen av en rad er nærmere beskrevet i metadata, men kan summeres opp slik:
+This dataset contains synthetic data describing recorded gender of group of persons.
```

Comment:
The term for kjønn is sex.
See https://www.ssb.no/en/befolkning/folketall/statistikk/befolkning

Reply:
ricky word since we dont have a word for both sex and gender in norwegian. Didn't see the ssb defintions, so will correct it 👍🏻